src/redux/reducers/forgotPasswordReducer.js

```diff
@@ -0,0 +1,49 @@
+import {
+  FORGOT_PASSWORD_FORM,
+  FORGOT_PASSWORD_SUCCESS,
+  FORGOT_PASSWORD_FAILURE,
+  CLEAR_FORGOT_PASSWORD_FORM,
+  SUBMIT_FORGOT_PASSWORD_FORM,
+} from '../actions-types/forgotPasswordTypes';
+
+const initialState = {
+  submitting: false,
+  success: false,
+  successMessage: '',
+  email: '',
+  newPassword: '',
+  confirmNewpassword: '',
+  message: '',
+  errors: [],
+};
+const forgotPasswordReducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case CLEAR_FORGOT_PASSWORD_FORM:
+      return { ...initialState };
+    case SUBMIT_FORGOT_PASSWORD_FORM:
+      return {
+        ...state,
+        submitting: payload.submitting,
+      };
+    case FORGOT_PASSWORD_FORM:
+      return {
+        ...state,
+        [payload.field]: payload.value,
+        errors: [],
+        message: '',
+      };
+    case FORGOT_PASSWORD_SUCCESS:
+      return { ...state, successMessage: payload.message, submitting: false };
```

Comment:
why is this return in a single line?

Reply:
@kimotho-njoki it was my auto format from VSCode putting it back to one line. I have disabled it and change to multilines